src/oom/oomd-util.h

```diff
@@ -76,7 +76,17 @@ static inline int compare_pgscan_and_memory_usage(OomdCGroupContext * const *c1,
         if (r != 0)
                 return r;
 
-        r = CMP((*c2)->pgscan, (*c1)->pgscan);
+        /* If one increased and the other decreased, sort the increased rate first */
+        r = CMP((*c2)->pgscan > (*c2)->last_pgscan, (*c1)->pgscan > (*c1)->last_pgscan);
```

Comment:
I see what you mean now. I confirmed with the kernel folks that lest there be a kernel bug or the cgroup was recreated pgscan should be monotonic. So I think what I want to do here is it (pgscan > last_pgscan), assume like last_pgscan was zero. and that the cgroup was recreated. But also log about it.

Reply:
Ohh, good point on the cgroup being possibly recreated!

Reply:
> In practice the kernel shouldn't overflow unless "70 zettabytes worth of memory" were scanned.

So it will be fine as long as one doesn't open more than 3 tabs in Chrome, in other words :-D